README.md

```diff
@@ -16,11 +16,18 @@ A config file is needed, whose contents should look like:
 
 ```json
 {
+	"DBType":     "sqlite3",
+	"DBPath":     "sqlite:///obmd.db",
```

Comment:
You still need to get rid of the `sqlite:` scheme here. Right now the behavior of this is to try to create a file at `$PWD/sqlite:/obmd.db`.

Reply:
How should the sqlite DB documentation be taken care of? I figured we should have it set like it is in HIL, where a `.db` file is created.  Or, alternatively, should I replace that example with postgres?

Reply:
Why not just change it to `./obmd.db`? this makes it fairly clear that it's a file path.

Reply:
That sounds good. I've updated it now.